static/templates/stream_settings/stream_types.hbs

```diff
@@ -13,6 +13,15 @@
     {{/each}}
 </div>
 
+<div id="announce-new-stream">
+    <label class="checkbox">
+        <input type="checkbox" name="announce" value="announce" checked />
+        <span></span>
+        {{t "Announce stream" }}
+    </label>
+    <span class="fa fa-question-circle settings-info-icon" aria-hidden="true" id="announce-stream-docs"></span>
+</div>
```

Comment:
I think this is going to create a bug in the stream-edit form, where this appears there even though it does nothing in that context. We probably need a conditional.

Reply:
Ugh, good catch.  I fixed it with conditionals.